webplugin/js/app/zendesk-chat-service.js

```diff
@@ -17,7 +18,7 @@ function ZendeskChatService() {
         var s = document.createElement("script");
         s.type = "text/javascript";
         s.async = true;
-        s.src = "https://dev.zopim.com/web-sdk/latest/web-sdk.js";
+        s.src = "https://km-prod-cdn.s3.us-east-1.amazonaws.com/kommunicate/zendesk-web-sdk-1.11.2.js";
```

Comment:
use .kommunicate.io url for CDN
Customers will allow only *.kommunicate.io in their firewall security policies

Reply:
done